### ISSUE TITLE
Temp sensor DHT_accessory.js memory problem

### ISSUE THREAD

**Issue by RickGC** [legotheboss/YouTube-files#21]
Hi, i was trying Run this file DHT_accessory.js, when I start core.js Run perfect but a few minutes latter crashes with the following message Cannot allocate memory
Failed to initialize sensor I tried some things but I can’t solve the problem even read this link https://github.com/momenso/node-dht-sensor/issues/6 and I was trying to understand how to fix it but I don’t get it yet, can you help me? Sorry for my English I from Mexico :)

**Comment by legotheboss** [comment]
Yeah, I am looking into this and going to try to solve it. It seems to work properly on some pis and gives membus errors on others.

**Comment by rudgat** [comment]
Hi, I see that this was listed in Sept 2017 and am having the same issue. Was this ever resolved?
